### PR TITLE
Overhaul to Ice River mission

### PR DESCRIPTION
**Pull request by OcelotWalrus**

### CODE REVIEW

**Comment by Dzierzan** [comment]
I like how it works now.

Add 3 or 4 machine gun pods for your base.

AI does not have any hardcap on mobile repair tanks, please fix it.

Does AI cheat when it comes to resources? I captured all Mining Towers and I felt like AI kept building stuff.

Add extra AI balloon for patrolling.

If /all cheat is used, the game might crash because actors like `mgpodpatrol` do not have sequences. I guess it's a leftover?

Extra stuff to do:
- Disable Ore Purifier/Ore Smelter
- Disable Elite Pod
- Disable Airstrike power.

**Comment by OcelotWalrus** [comment]
> Add 3 or 4 machine gun pods for your base.

What do you mean by that?

> Does AI cheat when it comes to resources? I captured all Mining Towers and I felt like AI kept building stuff.

Normally no. It's just that the AI had a lotta cash.

> Add extra AI balloon for patrolling.

There is already one.

> If /all cheat is used, the game might crash because actors like mgpodpatrol do not have sequences. I guess it's a leftover?

yes.

I'll update the PR later.

**Comment by Dzierzan** [comment]
> What do you mean by that?
I think that you should start the mission with 3 or 4 extra MG pods.

**Comment by OcelotWalrus** [comment]
The error is actually caused by the original rules. It fails because of the orepurifier:

```yaml
OREPURIFIER:
	Inherits: ORESMELT
	Buildable:  # first here
		Prerequisites: ~structures.yi, techcenter
	Selectable:
		Bounds: 1024, 2048, 0, 0
	Tooltip:
		Name: actor-orepurifier.name
	Buildable:  # duplicated
		Description: actor-orepurifier.description
	Encyclopedia:
		Description: actor-orepurifier.encyclopedia
	-WithIdleOverlay@Shadow:
	WithIdleAnimation:
		Interval: 35
```

**Comment by Mailaender** [comment]
https://github.com/OpenHV/OpenHV/commit/4412c2cdf7b15a911ec0c5b2da236f45f5074f32

**Comment by OcelotWalrus** [comment]
Is there anyway to remove a removal on a map rules.yaml?

Like remove a `-Trait:` trait?

File: mods/hv/maps/ice-river/rules.yaml [hunk 1]
```diff
@@ -3,6 +3,108 @@
...
 	PlayerResources:
 		DefaultCash: 8000
+	GrantConditionOnBotOwner@CampaignAI:
+		Condition: gentle-ai-enabled
+		Bots: campaign
+	BaseBotModule@CampaignAI:
+		RequiresCondition: gentle-ai-enabled
+	BuildingRepairBotModule:
+		RequiresCondition: gentle-ai-enabled
+	SquadManagerBotModule@CampaignAI:
```
[the remark this answers is not in the document]
Comment (OcelotWalrus) [reply mods/hv/maps/ice-river/rules.yaml:13]: Sure, I'll do that.

**Comment by OcelotWalrus** [comment]
Moved to #1055

**Pull request closed without merging**